run_dir/static/js/projects.js

```diff
@@ -129,13 +140,10 @@ function load_table_head(columns){
 }
 
 
-
-
-// Undefined columns handled here
-function load_undefined_columns() {
-  return $.getJSON("/api/v1/projects_fields?undefined=true&project_list=" + projects_page_type, function(data) {
+// Undefined columns handled here now
+function load_undefined_columns(cols) {
     var columns_html = "";
-    $.each(data, function(column_no, column) {
+    $.each(cols, function(col_id, column) {
       $("#undefined_columns").append('<div class="checkbox">'+
           '<label>'+
             '<input type="checkbox" class="filterCheckbox" data-columngroup="undefined-columns" data-displayname="'+column+'" name="'+column+'" id="undefined-columns-'+column+'">'+
```

Comment:
not using it, no, but $.each on an array gives out two members, if I only
have one, it's the index of the array, which is pretty useless

On Fri, Mar 20, 2015 at 4:07 PM, Guillermo Carrasco <
notifications@github.com> wrote:

> In run_dir/static/js/projects.js
> <https://github.com/SciLifeLab/genomics-status/pull/181#discussion_r26846456>
> :
>
> >      var columns_html = "";
> > -    $.each(data, function(column_no, column) {
> > +    $.each(cols, function(col_id, column) {
> >        $("#undefined_columns").append('<div class="checkbox">'+
> >            '<label>'+
> >              '<input type="checkbox" class="filterCheckbox" data-columngroup="undefined-columns" data-displayname="'+column+'" name="'+column+'" id="undefined-columns-'+column+'">'+
>
> not using, the col_id, right? Just checking that you didn't typo it with
> column
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/genomics-status/pull/181/files#r26846456>.
>